mainapp/templates/mainapp/ngo_form.html

```diff
@@ -20,4 +20,11 @@ <h2 class="text-center">വൊളന്‍റീയര്‍ ആകാന്‍<
   {% endbuttons %}
   </form>
 
+<script type="text/javascript">
+  window.addEventListener('load', function() {
+    $(function() {
```

Comment:
@thezillion  The `$()` is a shorthand for the `$.ready` which is not required here. As the **onLoad** event happens after **domReady**, you don't have to wrap the function inside a `$`

```js
$(function() {
```

You could even write the whole block like this:

```js
$(window).on('load', function(){
   $('#id_district').selectize();
})
```

Reply:
You're right. I've made the update. Removed the IIFE like call